sdks/python/apache_beam/utils/retry.py

```diff
@@ -57,6 +60,18 @@
 # pylint: enable=wrong-import-order, wrong-import-position
 
 _LOGGER = logging.getLogger(__name__)
+_RETRYABLE_REASONS = [
+    "rateLimitExceeded", "quotaExceeded", "internalError", "backendError"
+]
+_RETRYABLE_TYPES = (
```

Comment:
Should we be DRY here and avoid `if exceptions`? i.e.
```
_RETRYABLE_EXCEPTIONS_FROM_MODULE = [
    exceptions.TooManyRequests,
    exceptions.InternalServerError,
    exceptions.BadGateway,
    exceptions.ServiceUnavailable,
    exceptions.DeadlineExceeded
] if exceptions else []

_RETRYABLE_TYPES = tuple([
    *_RETRYABLE_EXCEPTIONS_FROM_MODULE,
    requests.exceptions.ConnectionError,
    requests.exceptions.Timeout,
    ConnectionError
])
```

Reply:
We have tests in this repo that run in environments where these libraries are not available. I'll rewrite it in a more clean way

Reply:
Actually, these aren't needed. There's logic elsewhere in this file that takes care of them